install.sh

```diff
@@ -7,7 +7,10 @@ if [ "$(uname)" != "Darwin" ]; then
   exit 1
 fi
 
-mkdir -p "$HOME"/.local/bin "$HOME"/.docker/cli-plugins
+# Create directories.
+[ ! -d "$HOME/.local/bin" ] && mkdir -p "$HOME/.local/bin"
```

Comment:
You don't really need the [ ! -d ] as mkdir will always return 0 even if the dir already exists.